great-idea/css/index.css

```diff
@@ -63,4 +63,181 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.container, .services-container {
+	width: 878px;
+	margin: auto;
+}
+
+/* header */
+	header {
+		display: flex;
+		align-items: center;
+		flex-wrap: nowrap;
+		height: 100%;
+		justify-content: space-around;
+		margin-top: 20px;
+	}
+	header nav {
+		display: flex;
+		align-items: flex-start;
+		flex-wrap: nowrap;
+		height: 100%;
+		margin-bottom: 25px;
+	}
+
+	header nav a {
+		color: gray;
+		text-decoration: none;
+		margin: 0 20px;
+	}
+
+	header img.logo {
+		margin-left: 50px;
+		margin-bottom: 25px;
+	}
+/* /header */
+
+/* index.html */
+
+	/* cta */
+		.cta {
+			display: flex;
+			flex-wrap: nowrap;
+			justify-content: space-around;
+			align-items: center;
+			margin-bottom: 50px;
+		}
+		.cta-text {
+			display: flex;
+			font-size: 70px;
+			text-align: center;
+			vertical-align: top;
+			padding: 55px;
+			margin-left: 25px;
+			margin-top: 30px;
+			flex-direction: column;
+			align-items: center;
+		}
+		.cta-text button {
+			font-size: 12px;
+			background: none;
+			box-shadow: none;
+			border-radius: 1px;
+			width: 120px;
+			height: 25px;
+		}
+		.circle-code {
+			display: flex;
+			vertical-align: top;
+			margin-left: 50px;
+			margin-top: 45px;
+		}
+	/* cta */
+
+	/* main-content */
+		div.top-content {
+			display: flex;
+			flex-flow: row nowrap;
+			align-content: flex-start;
+			justify-content: space-between;
+			padding: 20px 0px 20px 0px;
+		}
+		div.top-content div.text-content-a {
+			width: 49%;
+		}
+		div.top-content div.text-content-b {
+			width: 49%;
+		}
+
+	 	img.middle-img {
+			width: 100%;
+			display: flex;
+		}
+
+		div.bottom-content{
+			display: flex;
+			flex-flow: row nowrap;
+			align-content: flex-start;
+			justify-content: space-between;
+			padding: 20px 0px 20px 0px;
+			/* border: 1px solid green;*/
+		}
+		div.bottom-content div.text-content-0 {
+			width: 32%;
+		}
+		div.bottom-content div.text-content-1 {
+			width: 32%;
+		}
+		div.bottom-content div.text-content-2 {
+			width: 32%;;
+		}
+	/* main-content */
+
+	/* contact */
+		.contact {
+			line-height: 1.6;
+		}
+	/* contact */
+
+	/* footer */
+		footer {
+			text-align: center;
+			font-size: 12px;
+			margin-bottom: 80px;
+		}
+	/* footer */
+/* index.html */
+
+/* services.html */
+	.services-p {
+		padding: 20px 0px 20px 0px;
+	}
+	.services-p h2 {
+		font-size: 23px;
+	}
+
+	/* services-boxes */
+		section.services-boxes {
+			display: flex;
+			flex-wrap: wrap;
+			justify-content: space-between;
+			align-content: space-around;
+		}
+		div.service-box {
+			border: 1px solid black;
+			width: 48%;
+			margin: 15px 0px 15px 0px;
+			padding: 15px;
+			background-color: Gainsboro;
+		}
+		div.service-box button {
```

Comment:
The width of the border is pretty thick, so it might be a good idea to tone it down with `border-width` (_or alternatively use the `border` shortand, which is `border: <width> <style> <color>;